tox.ini

```diff
@@ -82,7 +82,7 @@ commands = flake8 --ignore=E402 --exclude="test" libcloud/
            flake8 --ignore=E402 integration/
            flake8 --ignore=E402,E902 docs/examples/
            flake8 --ignore=E402,E902 --max-line-length=160 contrib/
-           python -mjson.tool libcloud/data/pricing.json
+           python -mjson.tool libcloud/data/pricing.json /dev/null
```

Comment:
Is this line missing `>` for stdout redirect aka `> /dev/null`?

Reply:
Or does this syntax also work and second argument is file where to print / save the output?

Reply:
@Kami the second option! `>` does not appear to be supported by tox commands. See outfile in https://docs.python.org/3/library/json.html#json-commandline